### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by ollieatkinson**

### CODE REVIEW

**Review by paulsamuels — APPROVED** [review]
🐒

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+---
+
+**Describe the bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
```
Comment (Sherlouk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: It's a CLI tool, if we've got scrolling involved I would hope it's outside of the scope of this project 😂

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop:**
+ - OS Version: [e.g. iOS]
```
Comment (Sherlouk) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Is this super applicable? Or at least should be like `macOS 10.13` as the example.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+ - Guise Version [e.g. 1.0.0]
+
+**Additional context**
+Add any other context about the problem here.
```
Comment (Sherlouk) [inline .github/ISSUE_TEMPLATE/bug_report.md:24]: Extra relevant information might be the actual module they're trying to make a interface file for?

File: .github/ISSUE_TEMPLATE/feature_request.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+---
+
+**Is your feature request related to a problem? Please describe.**
+A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
+
+**Describe the solution you'd like**
+A clear and concise description of what you want to happen.
+
+**Describe alternatives you've considered**
+A clear and concise description of any alternative solutions or features you've considered.
```
Comment (Sherlouk) [inline .github/ISSUE_TEMPLATE/feature_request.md:14]: To confirm, clear and concise description? Said it a couple times

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (ollieatkinson) [reply .github/ISSUE_TEMPLATE/bug_report.md]: These were just generated by GitHub, and are meant to be examples - but you're right we should try make them more specific to the tool

Comment (Sherlouk) [reply .github/ISSUE_TEMPLATE/bug_report.md]: Yea should definitely customise them a little, quite a few weird statements which make no sense given the context!

**Pull request merged**